cmd/tealdbg/localLedger.go

```diff
@@ -53,6 +69,25 @@ func makeAppLedger(
 	apps := []basics.AppIndex{appIdx}
 	apps = append(apps, txn.Txn.ForeignApps...)
 
+	// first populate balances from the indexer
+	for _, acc := range accounts {
+		fmt.Println("debug hello")
+		if _, ok := balances[acc]; !ok {
+			queryString := fmt.Sprintf("%s/v2/accounts/%s?round=%v", indexerURL, acc, round)
+			fmt.Println(queryString)
+			if resp, err := http.Get(queryString); err == nil {
+				if responseData, err := ioutil.ReadAll(resp.Body); err == nil {
```

Comment:
Make sure to handle the error here. 

Also consider using something other than `ioutil.ReadAll`, which will try to process huge responses. If someone mistypes a URL, limiting the amount of response data read over the network will prevent `tealdbg` from trying to allocate tons of memory and then crashing.